src/ext/Bal/stdbas/WixStandardBootstrapperApplication.cpp

```diff
@@ -1231,34 +1231,52 @@ class CWixStandardBootstrapperApplication : public CBootstrapperApplicationBase
         __inout int* pResult
         )
     {
+        HRESULT hr = S_OK;
```

Comment:
I wondered if `DisplayFilesInUseDialogCondition` should be evaluated in `OnExecutePackageBegin` or maybe `OnPlanPackageBegin` instead.  I left it here because that's the closest to its only use right now.  Happy to move it if either of those are more appropriate places.